README.md

```diff
@@ -188,7 +188,7 @@ On Windows
 ```bash
 # Add these packages if torch.distributed is needed.
 # Distributed package support on Windows is a prototype feature and is subject to changes.
-conda install -y -q -c rdonnelly libuv
+conda install -c conda-forge libuv=1.39
```

Comment:
looking at anaconda cloud, conda-forge channel does not have libuv 1.39 for windows, but anaconda channel does. Or did I miss anything?

https://anaconda.org/conda-forge/libuv/files
https://anaconda.org/anaconda/libuv/files

Reply:
Check with @gunandrose4u offline. Looks like conda-forge would also query anaconda channel.